### PR TITLE
Suggested edits to address reviews of Chris' density branch

### PR DESCRIPTION
**Pull request by jcass11**
# Purpose
I took a stab at addressing the recommendations @chantelleleveille and I made in @cfrick13 's density branch PR. I am making a PR into his original branch with the hopes of having handled most of the edits needed to clear bandwidth for Chris. I completed this in a separate branch so that if I misinterpreted what Chris was trying to do or he doesn't like my edits is easy to change before affecting his branch.

# Edits
- Removed references to old density and old filtering
- Added the new workflows for panels of Fig 3 and SFig S4 to be called within the `figure_3_s4_workflow`, so they are all one workflow
- Import updates (removing unused imports and fixing broken ones, which also fixes the mypy test that was failing on the original density branch)
- Removed commented code
- Used the example tracks dictionary

# Testing
- ✅ Ran the new `figure_3_s4_workflow` first - which now includes all workflows related to these figures - which succeeded
- ✅ Ran `run_all_manuscript_workflows`

# What didn't I address here that was part of the original density PR?
I didn't touch removing the actual old denisty features themselves, just the references to them. There were also a huge number of new features added in the `density_fig_updates` PR and I didn't feel prepared to decide how to pate that down. So @cfrick13 that will still need to be done with updates to your PR not covered here.

### CODE REVIEW

**Review by chantelleleveille — APPROVED** [review]
Nice :) Thanks for doing that!

**Comment by chantelleleveille** [comment]
@cfrick13 @jcass11 If it would be easier to address newly added columns in its own PR after adding the new workflows I would be okay with that!

**Comment by jcass11** [comment]
> @cfrick13 @jcass11 If it would be easier to address newly added columns in its own PR after adding the new workflows I would be okay with that!

Fine with me!

File: test/lib/preprocessing/test_voronoi.py
Comment (cfrick13) [inline test/lib/preprocessing/test_voronoi.py:1]: this file was old and not added by me. I think it is used to validate that the voronoi tesselation is working so we may want to keep it?

**Review by cfrick13 — APPROVED** [review]
I think we may want to keep the `test_voronoi.py` file. This was not added by me. I think it is one of the tests written to validate the voronoi tesslation distance measurment which is useful for the toymodel.

File: test/lib/preprocessing/test_voronoi.py
Comment (jcass11) [reply test/lib/preprocessing/test_voronoi.py:1]: Added the function back!

**Pull request merged**